AGENTS.md

```diff
@@ -0,0 +1,187 @@
+# AGENTS.md
+
+This file provides guidance to AI code assistants when working with code in this repository.
+
+## Overview
+
+The Cluster Version Operator (CVO) is a core OpenShift Cluster Operator that manages cluster upgrades by reconciling release payload images to the cluster. It continuously monitors for available updates from the OpenShift Update Service (OSUS) and orchestrates the systematic rollout of new versions across all cluster operators.
+
+## Building and Testing
+
+### Build commands
+```bash
+# Build the CVO binary
+make build                # Runs hack/build-go.sh, outputs to _output/
+
+# Build the CVO container image
+./hack/build-image.sh     # Creates local image with git version tag
+
+# Push development image to a registry
+REPO=quay.io/yourname ./hack/push-image.sh
+
+# Format code
+make format              # Run go fmt on all packages
```

Comment:
We probably want to update `CONTRIBUTING.md` to include this information, or at least mention the fact that a Makefile exists.

Reply:
Regarding updating the `CONTRIBUTING.md` file - I plan to do that for the OTE efforts. We will have more contributors, and first PRs showcase the desire for us, as maintainers, to enhance the contributing experience.